packages/vendure-plugin-accept-blue/package.json

```diff
@@ -22,7 +22,7 @@
     "build": "rimraf dist && yarn generate && tsc",
     "start": "yarn ts-node test/dev-server.ts",
     "generate": "graphql-codegen",
-    "test": "vitest run --bail 1"
+    "test": "vitest run"
```

Comment:
Don't like the `--bail 1` option? (If it's not working, you might have to `yarn` in the root of the project).

Why I added it: Sometimes a test in the beginning fails, causing follow up tests to also fail. This made it hard for me to see what was the root cause.
The bail option stops any subsequent tests after one fails, so you know which one is the culprit